R/mle-finder.R

```diff
@@ -21,3 +21,76 @@ linear_BFGS <- function(design, outcome, noise_var = 1) {
   return(MLE)
 }
 
+logit_loglik <- function(coef, design, outcome) {
+
+  n <- length(outcome)
+  log_lik <- sum(sapply(1:n, function(i){
```

Comment:
-1

You can do this without the sapply (but good job using sapply instead of a for loop), e.g. with

`sum(outcome * log(prob) + (1 - outcome) * log(1 - prob))` where prob is the expit of design %*% Beta